addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/ExtensionOpenApi.java

```diff
@@ -345,7 +345,7 @@ public OpenApiResults importOpenApiDefinitionV2(
 
             List<String> errors =
                     importOpenApiDefinition(
-                            Json.pretty(openApi),
+                            new String(Files.readAllBytes(file.toPath())),
```

Comment:
It's this way to add support for multi file definitions, #2913, doing just this change breaks that.

Reply:
Any good examples to test this multi file definition schema? I tried with https://github.com/Paldom/swagger-multifile-example/tree/master and in any case this does not seem to work either, getting different errors.

Reply:
The codebase has tests for that, `ExtensionOpenApiTest.shouldImportMultiFileV*`.